DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsControllerTests.cs

```diff
@@ -172,13 +184,162 @@ public void AddRegistrationPromptSelectPrompt_loads_existing_temp_data()
+
+            // When
+            var result = registrationPromptsController.AddRegistrationPromptSelectPrompt(expectedPromptModel);
+
+            // Then
+            registrationPromptsController.TempData.Peek<AddRegistrationPromptData>().Should().NotBeNull();
+            registrationPromptsController.TempData.Peek<AddRegistrationPromptData>()!.SelectPromptViewModel.Should()
+                .BeEquivalentTo(expectedPromptModel);
+            result.Should().BeRedirectToActionResult().WithActionName("AddRegistrationPromptConfigureAnswers");
+        }
+
+        [Test]
+        public void AddRegistrationPromptConfigureAnswers_next_updates_temp_data()
+        {
+            // Given
+            var expectedPromptModel = new AddRegistrationPromptSelectPromptViewModel
+            {
+                CustomPromptId = 1,
+                Mandatory = true
+            };
+            var initialTempData = new AddRegistrationPromptData { SelectPromptViewModel = expectedPromptModel };
+            registrationPromptsController.TempData.Set(initialTempData);
+            var model = new RegistrationPromptAnswersViewModel
+            {
+                Answer = null,
+                OptionsString = "Test"
+            };
+            const string action = "next";
+
+            // When
+            var result = registrationPromptsController.AddRegistrationPromptConfigureAnswers(model, action);
+
+            // Then
+            registrationPromptsController.TempData.Peek<AddRegistrationPromptData>().Should().NotBeNull();
+            registrationPromptsController.TempData.Peek<AddRegistrationPromptData>()!.SelectPromptViewModel.Should()
+                .BeEquivalentTo(expectedPromptModel);
+            registrationPromptsController.TempData.Peek<AddRegistrationPromptData>()!.ConfigureAnswersViewModel.Should()
+                .BeEquivalentTo(model);
+            result.Should().BeRedirectToActionResult().WithActionName("Index");
+        }
+
+        [Test]
+        public void AddRegistrationPromptConfigureAnswers_add_configures_new_answer_and_updates_temp_data()
+        {
+            // Given
+            var model = new RegistrationPromptAnswersViewModel
+            {
+                Answer = "Answer",
+                OptionsString = "Test"
+            };
+            var expectedPromptModel = new AddRegistrationPromptSelectPromptViewModel
+            {
+                CustomPromptId = 1,
+                Mandatory = true
+            };
+            var initialTempData = new AddRegistrationPromptData
+                { SelectPromptViewModel = expectedPromptModel, ConfigureAnswersViewModel = model };
+            registrationPromptsController.TempData.Set(initialTempData);
+            var expectedViewModel = new RegistrationPromptAnswersViewModel
+            {
+                OptionsString = "Test\r\nAnswer"
+            };
+            const string action = "addPrompt";

```

Comment:
I think naming things "model/Viewmodel/PromptModel" is fairly generic and makes this test a bit difficult to follow. I'd probably group same viewmodels together (declare same types next to each other) and have more specific names, like configureAnswersModel/expectedConfigureAnswersModel/selectPromptModel